earthpy/tests/test_stack.py

```diff
@@ -18,6 +18,45 @@ def in_paths(basic_image_tif):
     return [basic_image_tif] * 4
 
 
+@pytest.fixture
+def in_paths_mismatch(basic_image_tif, basic_image_tif_2):
+    """ Input file paths for tifs of different sizes to stack. """
+    return [basic_image_tif] * 4 + [basic_image_tif_2]
+
+
+@pytest.fixture
+def in_paths_CRS_mismatch(basic_image_tif, basic_image_tif_CRS):
+    """ Input file paths for tifs of different sizes to stack. """
+    return [basic_image_tif] * 4 + [basic_image_tif_CRS]
+
+
+@pytest.fixture
+def in_paths_Affine_mismatch(basic_image_tif, basic_image_tif_Affine):
+    """ Input file paths for tifs of different sizes to stack. """
```

Comment:
dimensions? or dimensions (nrows, ncols)

Reply:
copy pasta. I'll update :)

Reply:
![copypasta-all-the-spaghetti-code](https://user-images.githubusercontent.com/7649194/59058106-21ad8e80-8859-11e9-8cf3-ddcaa02a1cfa.jpg)